packages/fluent-theme/src/fluent/styles/Checkbox.styles.ts

```diff
@@ -33,7 +31,7 @@ export const CheckboxStyles = (props: ICheckboxStyleProps): Partial<ICheckboxSty
         !checked && {
           selectors: {
             ':hover .ms-Checkbox-text': { color: palette.neutralDark },
-            ':hover .ms-Checkbox-checkmark': { color: NeutralColors.gray120 } // color not in the palette or semanticColors
+            ':hover .ms-Checkbox-checkmark': { color: palette.neutralSecondary }
```

Comment:
i don't know if you want to address this or do it in another PR, but if this style is already using neutralSecondary in the base component, then it is not needed here

Reply:
(and same with all over the place. it would be preferable to remove as many styles as possible to keep the bundle size regression as small as possible...)

Reply:
Good point on that, but this PR is to unblock the testing of the package by other teams so I might take another look at compared to the default styles with a separate PR. Or maybe over the weekend in this PR. Thanks.

Reply:
Cleaned as much as possible without breaking anything. Don't think it's worth the time going over again as will move them to default styles and will make sure there are no redundant styles plus don't think I can trim a sensible amount off to make a dent in bundle size.